docker/.env

```
@@ -22,6 +22,7 @@ PSQL_DOCKER_HOST=postgres
 PSQL_PASSWORD=
 PSQL_PORT=5432
 PSQL_USER=postgres
+REACT_APP_DISABLE_ANALYTICS=true
```

we should default this to `false`
```suggestion
REACT_APP_DISABLE_ANALYTICS=false
```